src/test/com/string/Converter.java

```diff
@@ -0,0 +1,41 @@
+package test.com.string;
+
+/**
+ * Створити клас Converter з методом convert який приймає текст,
+ * <p>
+ * прибирає з початку, з кінця і з середини пробіли,
+ * <p>
+ * робить все маленьким шрифтом, обертає текст навпаки, розділяє його навпіл
+ * (якщо не рівно то перша частина на 1 символ буде більшою) і виводить в консоль результат.
+ * <p>
+ * На вхід: “ Some other Day “, на виході: yadreh toemos
+ * На вхів: “     Do the riGht thinG”   , на виході: gnihtthg irehtod
+ */
+public class Converter {
+    private static String str1 = " Some other Day ";
+    private static String str2 = "      Do the riGht thinG";
+
+    public static void main(String[] args) {
+        converter(str1);
+        converter(str2);
+    }
+
+    public static void converter(String str) {
+        String s = str.replaceAll(" ", "");
+        StringBuilder builder = new StringBuilder(s.toLowerCase());
+        StringBuilder reverse = builder.reverse();
```

Comment:
it's nice to know the right tool for the specific job, no need in inventing your own tools.

Reply:
you can rename str to input and rename reverse to reversedInput